<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 212, 212);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>861</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="welcomeScreen">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 73, 94);
color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QPushButton" name="loadGame">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>350</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(249,166,2);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Load Game</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startGame">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>290</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 128, 14);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Start Game</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>231</width>
        <height>331</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Player</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Score</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>60</y>
        <width>231</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Pencil &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Producer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;HighScores&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mainWidget">
     <widget class="QLabel" name="inventory">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(233, 185, 110);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="save">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Public demand:</string>
      </property>
     </widget>
     <widget class="QLabel" name="wallet">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(15,157,88);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="upgradeApm">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>390</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(249,166,2);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Upgrade APM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="speedup">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>20</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>Speed Up</string>
      </property>
     </widget>
     <widget class="QLabel" name="wood">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>110</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="rateofApm">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>220</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>220</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Rate of APM:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="intelligenceLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>290</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Intelligence</string>
      </property>
     </widget>
     <widget class="QLabel" name="marketingLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Marketing:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="makePencil">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(128, 0, 0);color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>Make Pencil</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>110</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Wood:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buyGraphite">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>290</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(52, 73, 94);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Buy Graphite</string>
      </property>
     </widget>
     <widget class="QLabel" name="priceofWood">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>440</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 91, 83);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>861</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>70</y>
        <width>20</width>
        <height>201</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="priceofApm">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>440</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 99, 155);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>70</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Total pencils produced:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="unlockMarketing">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>290</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(6, 161, 11);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Unlock Marketing</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalinventory">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>70</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(233, 185, 110);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="load">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
     <widget class="QLabel" name="numberofApm">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>190</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buyWood">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>390</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 91, 83);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Buy Wood</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>190</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>APM's:</string>
      </property>
     </widget>
     <widget class="QToolButton" name="increasePencil">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QLabel" name="marketingLevel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buyApm">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>390</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 99, 155);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Buy APM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Pencils Produced:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>150</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Graphite:</string>
      </property>
     </widget>
     <widget class="QLabel" name="priceofGraphite">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>340</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(52, 73, 94);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Price of Pencil:</string>
      </property>
     </widget>
     <widget class="QLabel" name="priceofMarketingUpgrade">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 128, 14);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="pricetoUnlockMarketing">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>340</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(6, 161, 11);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="publicDemand">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="upgradeMarketing">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 128, 14);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Upgrade Marketing</string>
      </property>
     </widget>
     <widget class="QLabel" name="pencilPrice">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Bank Balance</string>
      </property>
     </widget>
     <widget class="QLabel" name="intelligence">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>330</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(15,157,88);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>cmr10</family>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:normal;&quot;&gt;Pencil Producer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QToolButton" name="decreasePencil">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>90</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="graphite">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>150</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="priceofApmUpgrade">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>440</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(249,166,2);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="debug">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>20</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Debug</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
     <zorder>upgradeMarketing</zorder>
     <zorder>buyWood</zorder>
     <zorder>buyApm</zorder>
     <zorder>inventory</zorder>
     <zorder>save</zorder>
     <zorder>label_7</zorder>
     <zorder>wallet</zorder>
     <zorder>upgradeApm</zorder>
     <zorder>speedup</zorder>
     <zorder>wood</zorder>
     <zorder>rateofApm</zorder>
     <zorder>label_12</zorder>
     <zorder>intelligenceLabel</zorder>
     <zorder>marketingLabel</zorder>
     <zorder>makePencil</zorder>
     <zorder>label_3</zorder>
     <zorder>buyGraphite</zorder>
     <zorder>priceofWood</zorder>
     <zorder>line_2</zorder>
     <zorder>line</zorder>
     <zorder>priceofApm</zorder>
     <zorder>label_11</zorder>
     <zorder>unlockMarketing</zorder>
     <zorder>totalinventory</zorder>
     <zorder>load</zorder>
     <zorder>numberofApm</zorder>
     <zorder>label_8</zorder>
     <zorder>increasePencil</zorder>
     <zorder>marketingLevel</zorder>
     <zorder>label_10</zorder>
     <zorder>label_4</zorder>
     <zorder>priceofGraphite</zorder>
     <zorder>label_5</zorder>
     <zorder>priceofMarketingUpgrade</zorder>
     <zorder>pricetoUnlockMarketing</zorder>
     <zorder>publicDemand</zorder>
     <zorder>pencilPrice</zorder>
     <zorder>label_6</zorder>
     <zorder>intelligence</zorder>
     <zorder>label</zorder>
     <zorder>decreasePencil</zorder>
     <zorder>graphite</zorder>
     <zorder>priceofApmUpgrade</zorder>
     <zorder>debug</zorder>
     <zorder>quit</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
